sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/__init__.py

```diff
@@ -4,66 +4,46 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from ._phone_number_administration_client import PhoneNumbersAdministrationClient
-from ._polling import ReservePhoneNumberPolling, PurchaseReservationPolling, ReleasePhoneNumberPolling
+from ._phone_numbers_client import PhoneNumbersClient
 
 from ._generated.models import (
     AcquiredPhoneNumber,
     AcquiredPhoneNumbers,
-    AreaCodes,
-    CreateSearchOptions,
-    CreateSearchResponse,
-    LocationOptionsQuery,
-    LocationOptionsResponse,
-    NumberConfigurationResponse,
-    NumberUpdateCapabilities,
-    PhoneNumberCountries,
-    PhoneNumberEntities,
-    PhoneNumberRelease,
-    PhoneNumberReservation,
-    PhonePlanGroups,
-    PhonePlansResponse,
-    PstnConfiguration,
-    ReleaseResponse,
-    UpdateNumberCapabilitiesResponse,
-    UpdatePhoneNumberCapabilitiesResponse
-)
-
-from ._shared.models import (
-    CommunicationUserIdentifier,
-    PhoneNumberIdentifier,
-    UnknownIdentifier
+    CommunicationError,
+    CommunicationErrorResponse,
+    PhoneNumberCapabilities,
+    PhoneNumberCapabilitiesRequest,
+    PhoneNumberCost,
+    PhoneNumberOperation,
+    PhoneNumberPurchaseRequest,
+    PhoneNumberSearchRequest,
+    PhoneNumberSearchResult,
+    BillingFrequency,
+    PhoneNumberAssignmentType,
+    PhoneNumberCapabilityValue,
+    PhoneNumberOperationStatus,
+    PhoneNumberOperationType,
+    PhoneNumberType,
 )
 
 __all__ = [
-    'PhoneNumbersAdministrationClient',
-    'ReservePhoneNumberPolling',
-    'PurchaseReservationPolling',
-    'ReleasePhoneNumberPolling',
-
-    # from _phonenumber
     'AcquiredPhoneNumber',
     'AcquiredPhoneNumbers',
-    'AreaCodes',
-    'CreateSearchOptions',
-    'CreateSearchResponse',
-    'LocationOptionsQuery',
-    'LocationOptionsResponse',
-    'NumberConfigurationResponse',
-    'NumberUpdateCapabilities',
-    'PhoneNumberCountries',
-    'PhoneNumberEntities',
-    'PhoneNumberRelease',
-    'PhoneNumberReservation',
-    'PhonePlanGroups',
-    'PhonePlansResponse',
-    'PstnConfiguration',
-    'ReleaseResponse',
-    'UpdateNumberCapabilitiesResponse',
-    'UpdatePhoneNumberCapabilitiesResponse',
-
-    # from _shared
-    'CommunicationUserIdentifier',
-    'PhoneNumberIdentifier',
-    'UnknownIdentifier'
+    'CommunicationError',
+    'CommunicationErrorResponse',
+    'PhoneNumberCapabilities',
+    'PhoneNumberCapabilitiesRequest',
```

Comment:
We don't expose "Request" and "Reponse" objects in the public API - what are these used for?